src/test/java/seedu/address/model/person/PhoneTest.java

```diff
@@ -8,17 +8,17 @@
 public class PhoneTest {
 
     @Test
-    public void isValidPhone() {
+    public void isValidPriority() {
         // invalid phone numbers
-        assertFalse(Phone.isValidPhone("")); // empty string
-        assertFalse(Phone.isValidPhone(" ")); // spaces only
-        assertFalse(Phone.isValidPhone("phone")); // non-numeric
-        assertFalse(Phone.isValidPhone("9011p041")); // alphabets within digits
-        assertFalse(Phone.isValidPhone("9312 1534")); // spaces within digits
+        assertFalse(Priority.isValidPriority("")); // empty string
+        assertFalse(Priority.isValidPriority(" ")); // spaces only
+        assertFalse(Priority.isValidPriority("phone")); // non-numeric
+        assertFalse(Priority.isValidPriority("9011p041")); // alphabets within digits
+        assertFalse(Priority.isValidPriority("9312 1534")); // spaces within digits
 
         // valid phone numbers
-        assertTrue(Phone.isValidPhone("93121534"));
-        assertTrue(Phone.isValidPhone("4")); // short phone numbers
-        assertTrue(Phone.isValidPhone("124293842033123")); // long phone numbers
+        assertTrue(Priority.isValidPriority("93121534"));
+        assertTrue(Priority.isValidPriority("4")); // short phone numbers
+        assertTrue(Priority.isValidPriority("124293842033123")); // long phone numbers
     }
```

Comment:
Yeah I will, I'm submitting the request so that you guys can merge and continue your work. I'll update the logic (regex related stuff) and test cases probably tomorrow. Thanks for reviewing!